### PR TITLE
Update DESTINATION_TEMPLATE.mdx

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

Updated destination template with latest standard and consistencies.

### CODE REVIEW

File: DESTINATION_TEMPLATE.mdx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "Destination_Name"
-description: Step-by-step guide to send your event data from RudderStack to **Destination_Name**.
+description: Step-by-step guide on sending your event data from RudderStack to **Destination_Name**.
+aliases: ["dest_name","dest_name_2"]
```
Comment (pfd) [inline DESTINATION_TEMPLATE.mdx:4]: Can you add a comment here about what aliases are? Something like

"Alias values are used to optimize the page in our search index."

**Review by pfd — APPROVED** [review]

File: DESTINATION_TEMPLATE.mdx (hunk 1 above)
Comment (ameypv-rudder) [inline DESTINATION_TEMPLATE.mdx:4]:
```suggestion
aliases: ["dest_name","dest_name_2"]  // These alias values are used to optimize the page our search index, making it easier for users to find the integration in the docs.
```

File: DESTINATION_TEMPLATE.mdx [hunk 2]
```diff
@@ -9,49 +10,99 @@ description: Step-by-step guide to send your event data from RudderStack to **De
...
 
-E.g.: "Google Analytics is the world's most popular analytics service that lets you track, analyze, and report your website traffic across a variety of sources."
+<div class="infoBlock">
+Find the open source transformer code for this destination in the <a href="https://github.com/rudderlabs/rudder-transformer/tree/master/v0/destinations/<dest_name>">GitHub repository</a>.
+</div>
 
 ## Getting started
 
-First, verify if the source platform is supported by \*\***Destination_Name**\*\* by referring to the following table:
+RudderStack supports sending events to <Destination_Name> via the following <a href="https://rudderstack.com/docs/rudderstack-cloud/rudderstack-connection-modes/">connection modes</a>:
```
Comment (ameypv-rudder) [inline DESTINATION_TEMPLATE.mdx:22]:
```suggestion
RudderStack supports sending events to <Destination_Name> via the following <Link to="/destinations/rudderstack-connection-modes/">connection modes</Link>:
```

[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply DESTINATION_TEMPLATE.mdx:22]: agreed - added!

File: DESTINATION_TEMPLATE.mdx (hunk 1 above)
Comment (ameypv-rudder) [reply DESTINATION_TEMPLATE.mdx:4]: makes sense, added this bit.

File: DESTINATION_TEMPLATE.mdx [hunk 3]
```diff
@@ -9,49 +10,99 @@ description: Step-by-step guide to send your event data from RudderStack to **De
...
 
-\*\*Add details about the configurable settings to set up your destination in RudderStack and where to get them, if applicable.\*\*
-
-\*\*For example:  **<Destination_Name> API Key**: Enter your <Destination_Name> API key here. You can obtain this key by logging into the <Destination_Name> console (provide URL to the console) and navigating to Settings - API key.**\*\*
-
-## Supported events (Track, Identify, Page, etc.)
-
-\*\*Add details for the events supported by the destination. Include relevant edge-cases, sample snippets, tips, etc. wherever required.\*\*
-
+To successfully configure Mixpanel as a destination, you need to configure the following settings:
```
Comment (ameypv-rudder) [inline DESTINATION_TEMPLATE.mdx:37]:
```suggestion
To successfully set up Mixpanel as a destination, you need to configure the following settings:
```

Comment (ameypv-rudder) [reply DESTINATION_TEMPLATE.mdx:37]: As "configure" is repeated.

**Pull request merged**